packages/blockchain-extension/extension/commands/UserInputUtil.ts

```diff
@@ -740,7 +740,7 @@ export class UserInputUtil {
 
     }
 
-    public static async browse(placeHolder: string, quickPickItems: string[] | { label: string, description: string }[], openDialogOptions: vscode.OpenDialogOptions, returnUri?: boolean): Promise<string | vscode.Uri | vscode.Uri[]> {
+    public static async browse(placeHolder: string, quickPickItems: string[] | { label: string, description: string }[], openDialogOptions: vscode.OpenDialogOptions): Promise<vscode.Uri | vscode.Uri[]> {
```

Comment:
Just noticed that we have to pass `openDialogOptions` through every time we call `browse`.
It would be nice if the function had some default options and we just passed flags through instead.